sim/druid/wrath.go

```diff
@@ -31,17 +33,18 @@ func (druid *Druid) registerWrathSpell() {
 			},
 
 			ModifyCast: func(_ *core.Simulation, _ *core.Spell, cast *core.Cast) {
+				//druid.applyNaturesGrace(cast)
 				druid.applyNaturesSwiftness(cast)
 			},
 		},
 
 		ApplyEffects: core.ApplyEffectFuncDirectDamage(core.SpellEffect{
 			ProcMask:             core.ProcMaskSpellDamage,
-			BonusSpellCritRating: 0,
-			DamageMultiplier:     1 + 0.02*float64(druid.Talents.Moonfury),
+			BonusSpellCritRating: float64(2 * float64(druid.Talents.NaturesMajesty) * 45.91),
+			DamageMultiplier:     1 + 0.02*float64(druid.Talents.Moonfury)*(1+0.01*float64(druid.Talents.ImprovedInsectSwarm)),
```

Comment:
I think you're missing some parenthesis () here. The moonfury term `0.02*float64(druid.Talents.Moonfury)` is not grouped with the `1` at the start so it won't get multiplied correctly.